stages/test/test_kickstart.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import os.path
+import shutil
+import unittest
+from tempfile import mkdtemp
+
+from org_osbuild_kickstart import main as kickstart_main
```

Comment:
We could do this by using `importlib` instead; note that it's a bit more code that we might want to share over multiple stage test cases and we might run into some issues compatibility wise (some things have changed over time in `importlib`) so I'm not going to press hard on this. It'd just be nicer :)

Reply:
Thanks! Indeed, my first impulse was importlib too, then I started reading the docs and got a bit discouraged and went with the symlink to illustrate my idea. But for the real thing(tm) using something nicer makes a ton of sense :)